src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul').children;
+
+function sortList(params) {
```

Comment:
The parameter name 'params' is not descriptive enough. Consider renaming it to 'listItems' to better describe its purpose.